rosidl_typesupport_c/CMakeLists.txt

```diff
@@ -109,6 +114,17 @@ if(BUILD_TESTING)
     OUTPUT
     "${TEST_LIB_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}rosidl_typesupport_c__test_type_support3${CMAKE_SHARED_LIBRARY_SUFFIX}"
     CONTENT "I'm not a shared library, why would you treat me like one?")
+
+  add_performance_test(benchmark_type_support_dispatch test/benchmark/benchmark_type_support_dispatch.cpp
+    APPEND_ENV
+    "DYLD_LIBRARY_PATH=${TEST_LIB_DIR}"
+    "LD_LIBRARY_PATH=${TEST_LIB_DIR}"
+    "PATH=${TEST_LIB_DIR}")
+  if(TARGET benchmark_type_support_dispatch)
+    target_include_directories(benchmark_type_support_dispatch PUBLIC include)
```

Comment:
These `target_include_directories` calls appearing in all of these performance tests aren't necessary. When you `target_link_libraries` against `${PROJECT_NAME}`, it's pulling in the whole interface, including the exported include directories of `${PROJECT_NAME}`.

Same goes for `rosidl_typesupport_cpp`.